bindings/gumjs/gumcmodule.c

```diff
@@ -382,12 +449,28 @@ gum_undecorate_name (const gchar * name)
 #endif
 }
 
-#else /* !HAVE_TINYCC */
+#endif /* HAVE_TINYCC */
+
+static void
+gum_cmodule_class_init (GumCModuleClass * klass)
+{
+}
+
+static void
+gum_cmodule_init (GumCModule * cmodule)
+{
+}
 
 GumCModule *
-gum_cmodule_new (const gchar * source,
+gum_cmodule_new (const gchar * name,
+                 const gchar * source,
                  GError ** error)
 {
+#ifdef HAVE_TINYCC
+  if (!name || strcmp (name, "tcc") == 0)
```

Comment:
```suggestion
  if (name == NULL || strcmp (name, "tcc") == 0)
```

To follow the current style of making all comparisons explicit. (Truthy checks should only be performed on booleans.)